upup/models/cloudup/resources/addons/networking.weave/k8s-1.7.yaml.template

```diff
@@ -203,6 +203,10 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: spec.nodeName
+            {{- if .Networking.Weave.NpcExtraArgs }}
+            - name: EXTRA_ARGS
+              value: "{{ .Networking.Weave.NpcExtraArgs }}"
+            {{- end }}
```

Comment:
There is actually no need to change this file as kops 1.18 doesn't support Kubernetes 1.7.

Reply:
I was hoping that we could get this in before kops 1.18. Is that no longer a possibility?

Reply:
I don't think Kubernetes 1.7 works on any of the recent versions of kops.